src/perf/dbserver/QuicPerformanceDataServer/Properties/serviceDependencies.MsQuicPerformanceResults - Web Deploy.json

```diff
@@ -0,0 +1,10 @@
+{
+  "dependencies": {
+    "mssql1": {
+      "resourceId": "/subscriptions/[parameters('subscriptionId')]/resourcegroups/[parameters('resourceGroup')]/providers/Microsoft.Sql/servers/quicperformanceresultsdbserver/databases/QuicPerformanceResults_db",
+      "type": "mssql.azure",
+      "connectionId": "DefaultConnection",
+      "secretStore": "AzureAppSettings"
+    }
+  }
+}
```

Comment:
Can we make GitHub happy and have all these files have a newline at the end?

Reply:
Done